utils/rpms/daos.spec

```
@@ -375,11 +375,11 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_libdir}/libdaos_common_pmem.so
 %{conf_dir}/vos_size_input.yaml
 %{_bindir}/daos_storage_estimator.py
-%{_libdir}/python3/site-packages/storage_estimator/*.py
-%dir %{_libdir}/python3/site-packages/storage_estimator
-%if (0%{?rhel} >= 7) && (0%{?rhel} < 8)
-%dir %{_libdir}/python3/site-packages/storage_estimator/__pycache__
-%{_libdir}/python3/site-packages/storage_estimator/__pycache__/*.pyc
+%{_libdir}/python3.6/site-packages/storage_estimator/*.py
```

Should we be using `python3_sitelib` here and everywhere else we have `python3.6` hardcoded: 
```suggestion
%{python3_sitelib}/storage_estimator/*.py
```

---

Ihad tried using %{python3_sitearch} and it failed I'll try sitelib